src/Microsoft.DotNet.Web.ProjectTemplates/content/WebApi-CSharp/Controllers/ValuesController.cs

```diff
@@ -24,7 +24,7 @@ public IEnumerable<string> Get()
 
         // GET api/values/5
         [HttpGet("{id}")]
-        public string Get(int id)
+        public string Get([FromRoute]int id)
```

Comment:
Space between `[FromRoute]` and `int id`

Reply:
Shouldn't need this, we should infer that it's `FromRoute` because it appears as a route value.